lib/web_ui/test/keyboard_test.dart

```diff
@@ -241,54 +241,6 @@ void main() {
 
       Keyboard.instance.dispose();
     });
-
-    test('ignores keyboard events triggered on text fields', () {
```

Comment:
Instead of deleting the test, let's modify it to expect that the keyboard events **_should_** be triggered on text fields.

Reply:
Good, OK.